### PR TITLE
feat: keras uses tree traversal

### PR DESCRIPTION
**Pull request by maximilianwerk**
Adapting `ImageKerasEncoder` for the new tree traversal already used in the encoders.

### CODE REVIEW

**Comment by JoanFM** [comment]
u need to bump manifest version for it to really activate tests and publish!

File: encoders/image/ImageKerasEncoder/config.yml [hunk 1]
```diff
@@ -10,8 +10,7 @@ requests:
     [IndexRequest, SearchRequest]:
       - !EncodeDriver
         with:
-          granularity_range: [1, 1]
```
Comment (JoanFM) [inline encoders/image/ImageKerasEncoder/config.yml:13]: I think Drivers should be removed by default, is this the only encoder with a Driver attached different than the default ones in the config.yml?

Comment (maximilianwerk) [reply encoders/image/ImageKerasEncoder/config.yml:13]: Yes it is.

Comment (JoanFM) [reply encoders/image/ImageKerasEncoder/config.yml:13]: So let's remove it, I think we should keep default config, any user can later adapt it, but for coherence as well I think Drivers should not be changed by default.

Comment (maximilianwerk) [reply encoders/image/ImageKerasEncoder/config.yml:13]: No, what I meant is, that it is the only executor in the hub, that explicitly does not need the default, yet (which is `['c', 'r']`.

Comment (JoanFM) [reply encoders/image/ImageKerasEncoder/config.yml:13]: I do not see why this should behave differently than ImageTorchEncoder?

**Review by rutujasurve94 — DISMISSED** [review]
LGTM

File: encoders/image/ImageKerasEncoder/config.yml (hunk 1 above)
Comment (JoanFM) [reply encoders/image/ImageKerasEncoder/config.yml:13]: And why shouldn't it be used to encode the root?

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**